kytos-init.sh

```diff
@@ -11,7 +11,7 @@ sed -i 's/WARNING/INFO/g' /etc/kytos/logging.ini
 
 test -z "$TESTS" && TESTS=tests/ 
 
-python -m pytest $TESTS
+python3 -m pytest $TESTS
 
 # only run specific test
 # python -m pytest --timeout=60 tests/test_e2e_10_mef_eline.py::TestE2EMefEline::test_on_primary_path_fail_should_migrate_to_backup
```

Comment:
```suggestion
# python3 -m pytest --timeout=60 tests/test_e2e_10_mef_eline.py::TestE2EMefEline::test_on_primary_path_fail_should_migrate_to_backup
```

Reply:
just applied the suggested commit, thanks @italovalcy